app/Models/User.php

```diff
@@ -147,7 +173,7 @@ public function purchasesWithoutRenewals(): HasMany
                 PurchasableType::TYPE_STANDARD_RENEWAL,
                 PurchasableType::TYPE_UNLIMITED_DOMAINS_RENEWAL,
             ]);
-        });
+        })->orWhereHas('bundle');
```

Comment:
Is this correct? Seems like all bundle buyers will be regarded as sponsors.

Reply:
This query doesn't have anything to do with sponsors? It retrieves all purchases without any renewals. So bundle purchases should be included

Reply:
Right! I was confused by the diff 🙂